package.json

```diff
@@ -28,6 +29,7 @@
   "homepage": "https://github.com/Polarisation/indexeddb-export-import#readme",
   "devDependencies": {
     "dexie": "^2.0.4",
+    "eslint": "^6.8.0",
```

Comment:
We should add eslint as a dev dependency in case the user didn't have it installed globally, and to ensure we are using the correct version